### ISSUE TITLE
Privacy page - accessibility - table columns headings should have attribute scope="col"

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#165]
fix by adding scope="col" to <td> of column headings

![Privacy_page_-_accessibility_-_table_columns_headings_should_have_attribute_scope=_col_](https://user-images.githubusercontent.com/2722522/65820581-e6c5d100-e222-11e9-96ab-01c3c0eb319e.png)
